lib/tasks/user.rake

```diff
@@ -1,12 +1,23 @@
 # frozen_string_literal: true
 
 namespace :user do
-  desc 'Updates the first_admin_id attribute value for all providers.'
+  desc 'Updates the first_admin_id attribute value for all accounts.'
   task update_all_first_admin_id: :environment do
     Account.transaction do
-      Account.providers.find_each do |account|
-        account.update!(first_admin_id: account.first_admin&.id)
-      end
+      complete_query = <<~SQL
```

Comment:
Does this run on Oracle?

Reply:
This is what I am trying to figure it out with CircleCI 😄